runners/metrics_client.go

```diff
@@ -60,13 +64,33 @@ func (c *prometheusClient) GetMetrics(ctx context.Context) (map[types.Namespaced
 		return nil, err
 	}
 
+	availableInodeSize, err := c.getMetricValues(ctx, inodesAvailableQuery)
+	if err != nil {
+		return nil, err
+	}
+
+	capacityInodeSize, err := c.getMetricValues(ctx, inodesCapacityQuery)
+	if err != nil {
+		return nil, err
+	}
+
 	for key, val := range availableBytes {
 		vs := &VolumeStats{AvailableBytes: val}
 		if cb, ok := capacityBytes[key]; !ok {
 			continue
 		} else {
 			vs.CapacityBytes = cb
 		}
+		if ais, ok := availableInodeSize[key]; !ok {
```

Comment:
In some systems, such as Btrfs, we cant get the inode size, in which case TopoLVM will not output the this metrics. 
ref. https://unix.stackexchange.com/questions/525637/debian-btrfs-filesystem-shows-a-total-of-zero-inodes-total-zero-used-zero-free/525650#525650
I think it is better to exec resize even if we can't get inode size.

Reply:
fixed it.
https://github.com/topolvm/pvc-autoresizer/pull/65/commits/285f063d1c10c1b3fc89abe84e56a4d5e8820bde